modules/@apostrophecms/area/ui/apos/components/AposAreaWidget.vue

```diff
@@ -92,6 +92,7 @@
           :tabbable="isHovered || isFocused"
           :menu-id="`${widget._id}-widget-menu-top`"
           :class="{[classes.open]: menuOpen === 'top'}"
+          :open="menuOpen === 'top'"
```

Comment:
Making this explicitly available as a prop so the menu can react to it and regenerate itself.